core-web/libs/block-editor/src/lib/extensions/image-tabview-form/components/dot-upload-tab/dot-upload-tab.component.scss

```diff
@@ -0,0 +1,40 @@
+@use "variables" as *;
+
+:host {
+    &,
```

Comment:
What is this for?

Reply:
I already changed this.

However, if you meant why I used `&`, it was to select the parent component. That way, we can apply the same style for the parent and child. It allows us to avoid writing the same style twice.